packages/app-page-builder/src/editor/plugins/elements/button/ButtonSettings.tsx

```diff
@@ -75,8 +75,11 @@ const ButtonSettings = ({
         postModifyElement: updateButtonElementIcon
     });
 
-    const updateType = useCallback(value => getUpdateValue("type")(value), [getUpdateValue]);
-    const updateIcon = useCallback(value => getUpdateValue("icon.id")(value?.id), [getUpdateValue]);
+    const updateType = useCallback((value: any) => getUpdateValue("type")(value), [getUpdateValue]);
+    const updateIcon = useCallback(
+        (value: PbIcon) => getUpdateValue("icon.id")(value?.id),
```

Comment:
How is value `undefined`? Your type says its always defined. No need to do `value?.id`.